recipes/jupyterlab-lsp/meta.yaml

```diff
@@ -0,0 +1,80 @@
+{% set name = "jupyter-lsp" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 3abe20ec48a3f6b132653abe597af2b6cd0005539ab0d74e0ba203927124d333
+
+build:
+  noarch: python
+  number: 0
+
+outputs:
+  - name: {{ name }}
+    script: build_base.bat  # [win]
+    script: build_base.sh  # [not win]
+    requirements:
+      host:
+        - python
+        - pip
+      run:
+        - notebook >=4.3.1
+        - python
+        - setuptools
+    test:  # Seems ignored by conda-build
+      requires:
+        - jupyterlab =2
+        - nodejs >=8,<13
+      commands:
+        - jupyter labextension install @krassowski/jupyterlab-lsp@^1.0.0
+        - python -m jupyterlab.browser_check
+  - name: {{ name }}-python
+    build:
+    requirements:
+      run:
+        - {{ pin_subpackage(name, exact=True) }}
+        - python-language-server
+    test:
+      files:
+        - modify_conftest.py
+      requires:
+        - jupyterlab ~=2.0
```

Comment:
proposed:
```
- jupyterlab >=2,<3.0.0a0
```
I think we want to use consistent pins for the lab versions, potentially hoisting that to a top-level jinja variable. _Thus far_ I haven't encountered any serious bugaboos vs the `2.1.x` line vs `2.0`... we need to get that back into the upstream test matrix.

Reply:
Done